axios_news_collector.py

```
@@ -0,0 +1,1132 @@
+                print(f"    Already in DB as: {source_url}")
+                action = "None" #"UPDATED"
+            else:
+                result = self.supabase.table('raw_news_collection').insert(article_data).execute()
+                action = "NEW"
+
+            if result.data:
+                record_id = result.data[0]['id'] if result.data else None
+                print(f"  [{action}] {news_item.company_name}: {news_item.funding_amount or news_item.valuation} - Score: {relevance_score}")
+                return record_id
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)